### ISSUE TITLE
Send "Intermediate Values" to backend

### ISSUE THREAD

**Issue by amgrubb** [amgrubb/BloomingLeaf#104]
Dependent on #57 and #93.

Implement SAT-based analysis for intermediate value table.
Figure out how to store and send values to the backend.

Question: How does this work for single path and all paths.

**Comment by amgrubb** [comment]
Created branch feature-userAssignments.

1. Add new class to backend ModelSpec

**Comment by amgrubb** [comment]
@g2hchang I need you to add the values from the table into InputEvaluation.js

User branch feature-userAssignments

lines 9--24, I left you some TODO:

```	//TODO, complete this function.
	//var value =  //get the values from the "Intermediate Values Table"
	//if(values != null){
		//for each value entered into the "Intermediate Values Table" {
			var goal = null;  //TODO
			var absTime = null; //TOOD
			var evaluationValue = null; //TODO
			var evaluation = new InputEvaluation(
					goal,
					absTime,
					evaluationValue);

			evaluations.push(evaluation);
	//	};
	//}

**Comment by oibrahim22** [comment]
We have not looked or changed anything in the backend this summer

**Comment by leshghi** [comment]
SPRINT WEEK OCT 1 @leshghi @epartakki 

**MUST DO:**
- [ ] Create sample model with 4 analysis values to use as an example/test model for Intermediate Values table @epartakki 
- [x] Narrow down all files relevant for Intermediate Values table and implementation in frontend @leshghi 

**SHOULD DO:**
- [ ] Find most recent place in code timeline where Intermediate Values table was working on frontend @leshghi 
- [ ] Compile list of questions for Alicia @epartakki @leshghi 
- [x] Narrow down all files relevant for Intermediate Values table and implementation in backend @epartakki @leshghi 

**STRETCH GOALS:**
- [x] Look for Intermediate Values table bugs
- [ ] Put together plan for how to fix Intermediate values table on frontend

**Comment by leshghi** [comment]
SPRINT WEEK OCTOBER 8 @leshghi @epartakki 
 
- [x] Create sample model with 4 analysis values to use as an example/test model for Intermediate Values table 
- [x] Look for Intermediate Values table bugs
- [x] Put together plan for how to fix Intermediate values table on frontend
- [x] Start working with code to debug frontend Intermediate Values table

**Comment by leshghi** [comment]
SPRINT WEEK OCTOBER 15 @leshghi 

- [x] Fix NT no drop down menu issue

- [x] Reorder numbers at proper timepoints

- [ ] Make list of all the major frontend bugs

**Comment by leshghi** [comment]
SPRINT WEEK OCTOBER 22 @leshghi 

- [x] Send values to backend

- [x] Investigate backend implementation

- [x] Continue debugging frontend

**Comment by leshghi** [comment]
SPRINT WEEK NOVEMBER 5 @leshghi 

- [x] Go over IVT with Alicia - figure out logic tweaks

- [ ] Implement correct logic in IVT

- [x] Fix minor bugs in develop found through debugger

- [ ] Familiarize with Ester's and Kathleen's issue in order to hit the ground running once finishing IVT major changes

**Comment by leshghi** [comment]
SPRINT WEEK NOVEMBER 12 @leshghi 

- [ ] Minor changes/updates to rearranging from analysis to modeling view

- [ ] Model management brainstorm

- [ ] IVT Logic
